sys-boot/gnu-efi/gnu-efi-3.0.6-r1.ebuild

```diff
@@ -61,8 +73,15 @@ efimake() {
 src_compile() {
 	tc-export BUILD_CC AR AS CC LD
 
-	# https://bugs.gentoo.org/607992
-	filter-mfpmath sse
+	if use custom-cflags; then
+		# https://bugs.gentoo.org/607992
+		filter-mfpmath sse
+
+		# https://bugs.gentoo.org/619628
+		append-flags -mno-avx
```

Comment:
This will break on non-x86 platforms, like ARM and IA64.
```
% aarch64-unknown-linux-gnu-gcc -mno-avx
aarch64-unknown-linux-gnu-gcc: error: unrecognized command line option ‘-mno-avx’; did you mean ‘-Wno-a,’?
```
Maybe replace it with this:
```
append-cflags $(test-flags-CC -mno-avx)
```